main.py

```diff
@@ -1,10 +1,58 @@
 import init_django_orm  # noqa: F401
 
+import json
```

Comment:
wrong import order(json should be imported first)

Reply:
Why?

Reply:
Your link does not want to open, send in telegram